package.json

```
@@ -7,23 +7,71 @@
   "icon": "icon.png",
   "license": "Apache-2.0",
   "engines": {
-    "podman-desktop": "^0.16.0"
+    "podman-desktop": ">=1.8.0"
   },
   "main": "./dist/extension.cjs",
   "source": "./src/extension.ts",
   "contributes": {
     "configuration": {
       "title": "Red Hat OpenShift Local",
       "properties": {
-        "OpenShift-Local.memory": {
+        "crc.factory.openshift.memory": {
           "type": "number",
           "format": "memory",
-          "description": "Memory size, in MiB"
+          "minimum": 11000000000,
+          "default": 11000000000,
+          "maximum": "HOST_TOTAL_MEMORY",
+          "step": 500000000,
+          "description": "Memory size, in GiB",
+          "scope": [
+            "KubernetesProviderConnectionFactory",
+            "DEFAULT"
+          ],
```

This does not work for me. Meaning I still see empty form when creating new CRC connection. I can see all the settings only if I change it to string property like this
```suggestion
          "scope": "KubernetesProviderConnectionFactory",
```

---

This PR requires https://github.com/containers/podman-desktop/pull/7659 to work.